packages/wrangler/src/api/dev.ts

```diff
@@ -0,0 +1,39 @@
+import { startDev } from "../dev";
+import { logger } from "../logger";
+interface DevOptions {
+	env?: string;
+	ip?: string;
+	port?: number;
+	localProtocol?: "http" | "https";
+	assets?: string;
+	site?: string;
+	siteInclude?: string[];
+	siteExclude?: string[];
+	nodeCompat?: boolean;
+	experimentalEnableLocalPersistence?: boolean;
+	_: (string | number)[]; //yargs wants this
+	$0: string; //yargs wants this
+}
+
+export async function unstable_dev(script: string, options: DevOptions) {
+	logger.warn(
+		`unstable_dev() is experimental\nunstable_dev()'s behaviour will likely change in future releases`
+	);
+
+	return new Promise<{ stop: () => void }>((resolve) => {
+		//lmao
```

Comment:
lmao is now part of the spec 

congrats on getting this to work, let's land it and see if we can simplify it in another pr